rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/BakeManifestEnvironment.java

```diff
@@ -56,4 +63,52 @@ private Path checkPath(final Path path) {
     }
     return path;
   }
+
+  /**
+   * Download an artifact that's a compressed tarball, and extract the contents of the tarball into
+   * this environment
+   */
+  public void downloadArtifactTarballAndExtract(
+      @NotNull ArtifactDownloader artifactDownloader, @NotNull Artifact artifact)
```

Comment:
We tend to use `java.annotation.Nonnull` here; no good reason I'm aware of other than consistency.

Reply:
Whoops, just noticed `@ParametersAreNonnullByDefault` [above](https://github.com/spinnaker/rosco/blob/e5fdff9e18f37029808efd2fb81adc17ecd07dcb/rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/BakeManifestEnvironment.java#L27).  Removed `@NotNull` here: https://github.com/spinnaker/rosco/pull/740/commits/c87e4d47acfca9b35b7772e3bbb976687282062d